engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java

```diff
@@ -147,12 +152,14 @@ private static final class Request {
         Output output;
         String nextToken;
         boolean last;
+        String seed;
 
-        Request(Input input) {
+        Request(Input input, String seed) {
             this.input = input;
             data = new ChunkedBytesSupplier();
             output = new Output();
             output.add(data);
+            this.seed = seed;
```

Comment:
why seed is a string?

Reply:
For serialization and deserialization.

Reply:
then this is not a seed? Why we need seed for serialize and deserialize

Reply:
uuid4